packages/reports/addon/models/deliverable-item.js

```diff
@@ -4,7 +4,22 @@
  */
 
 import DS from 'ember-data';
+import { A as arr } from '@ember/array'
+import { computed, get } from '@ember/object'
 
 export default DS.Model.extend({
-  deliveryRules: DS.hasMany('deliveryRule', { async: true, inverse: 'deliveredItem' })
+  deliveryRules: DS.hasMany('deliveryRule', { async: true, inverse: 'deliveredItem' }),
+
+  /**
+   * @property {DS.Model} deliveryRuleForUser - delivery rule model
+   */
+  deliveryRuleForUser: computed('user', 'deliveryRules.[]', function () {
```

Comment:
Can we also add a test for this for the dashboard model? Or move the test to the parent model?